examples/ChatQnA/ChatQnA_Guide.rst

```diff
@@ -3,89 +3,182 @@
+and without the GenAI Management Console (GMC), as well as cloud-native
+deployments using Red Hat OpenShift Container Platform (RHOCP).
+
+Preview
 =======
 
-AI catalog if applicable, or recorded demos. 
+To get a preview of the ChatQnA example, visit the
+`AI Explore site <https://aiexplorer.intel.com/explore>`_. The **ChatQnA Solution**
+provides a basic chatbot while the **ChatQnA with Augmented Context**
+allows you to upload your own files in order to quickly experiment with a RAG
+solution to see how a developer supplied corpus can provide relevant and up to
+date responses.
 
-Key Implementation Details 
+Key Implementation Details
 ==========================
 
-Tech Overview
-*************
+Embedding:
+  The process of transforming user queries into numerical representations called
+  embeddings.
+Vector Database:
+  The storage and retrieval of relevant data points using vector databases.
+RAG Architecture:
+  The use of the RAG architecture to combine knowledge bases and generative
+  models for development of chatbots with relevant and up to date query
+  responses.
+Large Language Models (LLMs):
+  The training and utilization of LLMs for generating responses.
+Deployment Options:
+  production ready deployment options for the ChatQnA
+  example, including single-node deployments and Kubernetes deployments.
 
 How It Works
 ============
 
-High level graphics to summarize the application.
+The ChatQnA Examples follows a basic flow of information in the chatbot system,
+starting from the user input and going through the retrieve, analyze, and
+generate components, ultimately resulting in the bot's output.
+
+.. figure:: /GenAIExamples/ChatQnA/assets/img/chatqna_architecture.png
+   :alt: ChatQnA Architecture Diagram
+
+   This diagram illustrates the flow of information in the chatbot system,
+   starting from the user input and going through the retrieve, analyze, and
+   generate components, ultimately resulting in the bot's output.
+
+The architecture follows a series of steps to process user queries and generate responses:
+
+1. **Embedding**: The user query is first transformed into a numerical
+   representation called an embedding. This embedding captures the semantic
+   meaning of the query and allows for efficient comparison with other
+   embeddings.
+#. **Vector Database**: The embedding is then used to search a vector database,
+   which stores relevant data points as vectors. The vector database enables
+   efficient and semantic retrieval of information based on the similarity
+   between the query embedding and the stored vectors.
+#. **Relevant Data**: The vector database retrieves the most relevant data
+   points based on the query embedding. These data points can include documents,
+   articles, or any other relevant information that can help generate accurate
+   responses.
+#. **LLMs**: The retrieved data points are then passed to large language models
```

Comment:
Should this be singular to match the fact that our pipeline currently uses just one LLM.

Reply:
We have a mix of nouns and verbs .. guess that is OK, the noun is LLM< the verb/action is creating response.